src/Chapter4.hs

```diff
@@ -259,7 +259,7 @@ name.
 
 > QUESTION: Can you understand why the following implementation of the
   Functor instance for Maybe doesn't compile?
-
+Maybe because in `fmap _ x = x` returns Maybe a insteed of Maybe b
```

Comment:
That's absolutely correct :+1: Function requires to return the value of type `Maybe b`, but you have value of type `Maybe a`